idaes/models_extra/column_models/tests/test_plate_heat_exchanger.py

```diff
@@ -14,15 +14,18 @@
 Tests for Plate Heat Exchnager  unit model.
 Author: Akula Paul
 """
-
+import sys
```

Comment:
If the change was reverted is this import still necessary?

Reply:
No, it is not.
```suggestion
```